packages/material-ui/src/Button/Button.js

```diff
@@ -2,9 +2,9 @@ import * as React from 'react';
 import PropTypes from 'prop-types';
 import clsx from 'clsx';
 import { unstable_composeClasses as composeClasses } from '@material-ui/unstyled';
+import { alpha } from '@material-ui/system';
```

Comment:
Not sure if it will have any impact.. The `styled()` util already depends on `styleFunctionSx` which basically composes the whole system, so not sure if it is worthed..

Reply:
I think if the `system` is not that big, I would rather use from root.
`import { alpha } from '@material-ui/system'` is more convenient.

Reply:
@siriwatknp I think that we could have a different policy for what we document and what we use internally: #24147.